src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -39,6 +41,7 @@ public class EditCommand extends Command {
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
             + "[" + PREFIX_NAME + "NAME] "
+            + "[" + PREFIX_AGE + "AGE ]"
```

Comment:
I think "AGE]" would be more correct here.

Reply:
Thanks, will rectify!